clients/src/main/java/org/apache/kafka/common/record/DefaultRecordBatch.java

```diff
@@ -158,7 +159,7 @@ public void ensureValid() {
 
     /**
      * Gets the base timestamp of the batch which is used to calculate the record timestamps from the deltas.
-     * 
+     *
```

Comment:
could you please remove this unrelated change

Reply:
Sorry for the delay as I was on vacation. I have deleted the unrelated changes.